### PR TITLE
[FIX] Clarify that electrodes.tsv is optional for MEG, for use with simultaneous (i)EEG

### PR DESCRIPTION
**Pull request by effigies**
Fixes https://github.com/bids-standard/bids-specification/issues/1550.

### CODE REVIEW

**Review by robertoostenveld — APPROVED** [review]
I approve.

**Review by sappelhoff — CHANGES_REQUESTED** [review]
(placing this here as a visual block, as this is not yet ready IMHO)

File: src/schema/rules/checks/meg.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Rules for MEG data that cannot be defined in tables
+---
+MEGElectrodesRecommended:
```
Comment (sappelhoff) [inline src/schema/rules/checks/meg.yaml:3]: as outlined in https://github.com/bids-standard/bids-specification/issues/1550#issuecomment-1686032272, I think this should be optional --> this electrode information **can** also be stored in `_headshape.<ext>` files or sometimes in the data file format itself (Elekta/Neuromag/MEGIN).

File: src/modality-specific-files/magnetoencephalography.md [hunk 2]
```diff
@@ -98,8 +98,10 @@ it SHOULD be stored separately under a new `/eeg` data type
 If however EEG is recorded simultaneously **with the same MEG system**,
 it MAY be stored under the `/meg` data type.
 In that case, it SHOULD have the same sampling frequency as MEG (see `SamplingFrequency` field below).
-Furthermore, the EEG sensor coordinates SHOULD be specified using MEG-specific coordinate
-systems (see [coordinates section](#coordinate-system-json-_coordsystemjson) below and
+Furthermore, EEG sensor coordinates SHOULD be recorded in an
+[`electrodes.tsv`](electroencephalography.md#electrodes-description-_electrodestsv)
+file using MEG-specific coordinate systems
+(see [Coordinate System JSON](#coordinate-system-json-_coordsystemjson) below and
```
Comment (sappelhoff) [inline src/modality-specific-files/magnetoencephalography.md:104]: this whole section should be rephrased to account for simultaneous recording with **i**EEG as well.

Simple adjustments like:

```diff
-Recording EEG simultaneously with MEG
+Recording (i)EEG simultaneously with MEG
```

etc.

File: src/modality-specific-files/magnetoencephalography.md [hunk 3]
```diff
@@ -98,8 +98,10 @@ it SHOULD be stored separately under a new `/eeg` data type
 If however EEG is recorded simultaneously **with the same MEG system**,
 it MAY be stored under the `/meg` data type.
 In that case, it SHOULD have the same sampling frequency as MEG (see `SamplingFrequency` field below).
-Furthermore, the EEG sensor coordinates SHOULD be specified using MEG-specific coordinate
-systems (see [coordinates section](#coordinate-system-json-_coordsystemjson) below and
+Furthermore, EEG sensor coordinates SHOULD be recorded in an
```
Comment (sappelhoff) [inline src/modality-specific-files/magnetoencephalography.md:101]: IMHO the SHOULD should become a MAY.

So far, the spec allowed for recording electrode positions in the `_headshape`: https://bids-specification.readthedocs.io/en/stable/glossary.html#headshape-suffixes

**Review by sappelhoff — COMMENTED** [review]
What do you think about this?

> perhaps we should add a section about `electrodes.tsv` to the [MEG section of the BIDS specification](https://bids-specification.readthedocs.io/en/latest/modality-specific-files/magnetoencephalography.html), and make corresponding schema adjustments.

- https://github.com/bids-standard/bids-specification/issues/1550#issuecomment-1643896148

I see that you have opted for not adding a new section and instead adding this information under `Recording EEG simultaneously with MEG`. I think I'd be fine with that as well.

**Review by sappelhoff — APPROVED** [review]

**Pull request merged**